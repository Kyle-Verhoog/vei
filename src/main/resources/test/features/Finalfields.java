// FAIL
// Java 1.3
public class Finalfields {
    public Finalfields() {}
    public final int x = 42;
}