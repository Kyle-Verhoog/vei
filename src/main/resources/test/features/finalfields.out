PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
FINAL ┃ final
INT ┃ int
IDENTIFIER ┃ x
= ┃ =
INTEGER_LITERAL ┃ 42
; ┃ ;
} ┃ }